### ISSUE TITLE
calico images not available for armv7

### ISSUE THREAD

**Issue by brunnels** [raspbernetes/k8s-cluster-installation#119]
Currently there are no calico container images for armv7l.  Maybe document this and add a pre-flight check?

**Comment by xunholy** [comment]
Yep, this is a good idea - we'll likely need to document to users to use arm64 architecture
